titus-server-master/src/main/java/com/netflix/titus/master/eviction/service/quota/job/UnhealthyTasksLimitTracker.java

```diff
@@ -42,10 +42,17 @@
 
 public class UnhealthyTasksLimitTracker implements QuotaTracker {
 
+    /**
+     * Do not track health status of small jobs with size below the threshold.
+     */
+    private static final int JOB_SIZE_TRACKING_THRESHOLD = 3;
```

Comment:
Can you change this to 2 so the only assumption on not highly available deployments is 1 instance?

Reply:
Done